eos/b-decays/observables.cc

```diff
@@ -56,6 +57,32 @@ namespace eos
 
         return ObservableGroup(imp);
     }
+
+    ObservableGroup
+    make_b_to_3l_nu_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Observables in $B^-\to \ell_\prime^+\ell_\prime^-\ell^-\bar\nu$ decays)",
+            R"(The option "lprime" selects the lepton flavour coming out of
+            the virtual photon, "l" selects the charged lepton flavour coming
```

Comment:
Please remove this explanation, since ``l`` and ``lprime`` should not be set by the user.